content/docs/explanation/netzgrafik-editor/_index.en.md

```diff
@@ -7,75 +7,35 @@ description: "Open-source software developed by SBB CFF FFS and its integration
-- a `standalone` flag:
-  - to tell NGE to disable all back-end interactions (no database on NGE side)
-  - to tell NGE to disable some UI components (authentication, project management, versioning system view, etc.)
-- implementation of a communication interface between OSRD and NGE ([`message_event`](https://developer.mozilla.org/fr/docs/Web/API/Window/message_event), [`eventListener`](https://developer.mozilla.org/fr/docs/Web/API/EventTarget/addEventListener) and [`postMessage`](https://developer.mozilla.org/fr/docs/Web/API/Window/postMessage)).
+For external use, a [build of NGE is available on NPM](https://www.npmjs.com/package/netzgrafik-frontend) and is published at each release. In addition to this, OSRD has developed a `standalone` version of NGE, integrated into the source code, which allows NGE to function without a back-end. Finally, to meet needs specific to OSRD, OSRD uses a [fork](https://github.com/osrd-project/netzgrafik-editor-frontend) of NGE (whose [build is also available on NPM](https://www.npmjs.com/package/@osrd-project/netzgrafik-frontend)), remaining as close as possible to the official directory.
 
-NGE is then able to get the OSRD timetable as soon as a change is made on OSRD side, and OSRD is able to get the changes made on NGE side.
+Despite the non-compatibility of the front-end technologies (ReactJS for OSRD and Angular for NGE), this build allows OSRD to integrate NGE within an [`iframe`](https://developer.mozilla.org/fr/docs/Web/HTML/Element/iframe). This `iframe` deploys a minimalist Angular application which then imports the real NGE application thanks to the imported build.
 
-The following diagram shows the workflow between the components.
+An alternative solution to the integration problem would have been to rewrite NGE as [`web-components`](https://developer.mozilla.org/fr/docs/Web/API/Web_components), in order to import them into OSRD, but this solution was abandoned because of the amount of work it would represent.
 
-![Concept diagram](osrd_nge_concept_diagram.jpg)
+NGE, in its `standalone` version, communicates with OSRD through the `iframe` using [`Operations`](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/main/documentation/STANDALONE.md). These messages are sent via DOM element properties:
+- [`@Input`](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/fe3e788499e18054e260c05e714419aeeafc44e1/src/app/app.component.ts#L75): with the `netzgrafikDto` property, triggered when the content of the scenario is updated from OSRD.
```

Comment:
That one doesn't use `Operations`. Maybe the `Operations` reference should be folded into the second list item.